status-im/status-react
# Implement Wallet Graph component

fixes #16739 #16705 

### Summary

Integrate `react-native-gifted-charts` library and implement Wallet Graph component

#### Results

https://github.com/status-im/status-mobile/assets/18485527/8dfd2b7b-c82f-415c-861b-219e60d1c435

https://github.com/status-im/status-mobile/assets/18485527/dbbeb12f-56d3-4b3d-8131-6f9829e547b2

#### Platforms

- Android
- iOS

#### Areas that maybe impacted

### Steps to test

- Open Status
- Log in
- Go to Settings > Quo2 Preview
- Check `wallet-graph` component under `Graph` section

status: ready
